File: 0136-single-number/0136-single-number.py
class Solution(object):
    def singleNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        
        for idx in range(len(nums)):
            value = nums.pop(idx)
            if value not in nums:
                return value
            else:
                nums.insert(idx, value)
        

File: 0136-single-number/0136-single-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var singleNumber = function(nums) {
    if(nums.length == 1) return nums[0]
    let count = {}
    for(let i=0; i<nums.length; i++){
        count[`${nums[i]}`] = count[`${nums[i]}`]+1 || 1
    }
    for(key in count){
        if(count[key] == 1){
            return key
        }
    }
};